sw-manifest.js

```diff
@@ -59,20 +60,15 @@ async function buildCacheManifest() {
     config.globPatterns.push('bootstrap.js', 'app.css');
   }
 
-  const toplevelManifest = await getManifest(config);
+  const toplevelManifest = await injectManifest(config);
```

Comment:
You might want to log the `toplevelManifest.count` value somewhere, which is the number of items precached. That was previously being logged, but you remove that message.

The properties available in the return value is documented at https://developers.google.com/web/tools/workbox/reference-docs/latest/module-workbox-build#.injectManifest

Reply:
added, thanks!